### PR TITLE
docs: move troubleshooting plugin loading

### PR DESCRIPTION
**Pull request by sympatheticmoose**
Makes it easier to discover and reference in links

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by sympatheticmoose** [comment]
Just a thought since a question popped up in slack and @marefr basically gave this advice 😄

File: docusaurus/docs/introduction/plugin-lifecycle.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Use of the Grafana CLI to install or uninstall a plugin requires you to restart
 
 Plugins are loaded either when Grafana starts up or when a plugin has been installed/uninstalled during runtime. 
 
-Understanding the different phases involved when Grafana is loading a plugin may help you better understand plugin usage and troubleshoot any unexpected behavior, such as why a certain plugin is not marked as installed in the plugins catalog and/or for use within Grafana even though you've installed it. You can check the [Grafana server log](https://grafana.com/docs/grafana/latest/troubleshooting/#troubleshoot-with-logs) for any unexpected errors or details related to loading a plugin. In addition, you can enable even more details by changing the [log level to debug](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#log).
+Understanding the different phases involved when Grafana is loading a plugin may help you better understand plugin usage and [troubleshoot](#troubleshooting) any unexpected behavior, such as why a certain plugin is not marked as installed in the plugins catalog and/or for use within Grafana even though you've installed it.
```
Comment (josmperez) [inline docusaurus/docs/introduction/plugin-lifecycle.md:37]:
```suggestion
Understanding the different phases involved when Grafana is loading a plugin may help you better understand plugin usage and [troubleshoot](#troubleshooting) any unexpected behavior. For example, why a certain plugin is not marked as installed in the plugins catalog or for use within Grafana even though you've installed it. 
```

**Review by josmperez — APPROVED** [review]
Good restructuring. I have made one suggestion to get sentence language < 25 word max.

File: docusaurus/docs/introduction/plugin-lifecycle.md (hunk 1 above)
Comment (sympatheticmoose) [reply docusaurus/docs/introduction/plugin-lifecycle.md]: Good suggestion. Applied

**Pull request merged**